plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -5133,6 +5133,10 @@ img.help_tip {
 
 			p.description {
 				margin-bottom: 8px;
+
+				&.is-error, span.is-error {
```

Comment:
I don't see a span tag heere, maybe just `&.is-error` is enough?
![image](https://github.com/woocommerce/woocommerce/assets/15019298/5624ddc1-86e8-44b1-a257-35c69b3de567)

Reply:
But `is-error` is currently applied on `<p>` tag, do you mean for the future use-case we are adding span here?

Reply:
Or maybe you meant, `p.is-error`?

Reply:
Yes, `is-error` is currently applied on the `p.description` tag if you want the full message to be error colored, but what if you want the description to render normally except for a specific part.

Say, if you create a setting with a description like `This is my description. <span class="is-error">This I want to highlight as an error.</span> But then the rest is normal.`

The (S)CSS above would let you do that.

Reply:
Oh, got it! thanks for explaining.